src/dymaptic.GeoBlazor.Core/Components/Widgets/Widget.cs

```diff
@@ -36,6 +36,14 @@ public abstract class Widget : MapComponent
     [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
     public string? ContainerId { get; set; }
 
+    /// <summary>
+    /// Icon which represents the widget. It is typically used when the widget is controlled by another one (e.g. in the Expand widget).
+    /// <a href="https://developers.arcgis.com/calcite-design-system/icons/">Calcite Icons</a>
+    /// </summary>
+    [Parameter]
+    [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
+    public string? Icon { get; set; }
```

Comment:
Does this need to be in the `MeasurementWidget.cs`? I saw that the both have Icon properties.  Just wanted to verify.

Reply:
it should be removed from the measurement widget and inherited from the widget class